src/OpenColorIO/ops/Truelight/TruelightOp.cpp

```diff
@@ -61,21 +61,27 @@ OCIO_NAMESPACE_ENTER
                         TransformDirection direction);
             virtual ~TruelightOp();
             
-            virtual OpRcPtr clone() const;
+            OpRcPtr clone() const override;;
```

Comment:
double ";" after some of the `override` additions in this file.

Reply:
Fixed.